### ISSUE TITLE
Able to Download External Subtitles

### ISSUE THREAD

**Issue by David-Moreira** [jellyfin/jellyfin-web#5618]
Hello,

I'd like to work on this feature 
https://features.jellyfin.org/posts/1181/subtitles-download

I believe just adding a download button by each external subtitle should be enough
![Untitled](https://github.com/jellyfin/jellyfin-web/assets/22283161/c5020343-6dc8-4184-84a1-a4476b8f155e)

It seems like the SubtitlesController might already support downloading the subtitle so no changes would need to be done on the server-side.
![image](https://github.com/jellyfin/jellyfin-web/assets/22283161/dc9add54-6c69-4cb0-9dbc-c14f1fac704f)

I'm new to the code base so I might be missing something. Would you guys give me some feedback on this and let me know whether it is ok that I work on this feature?

**Comment by thornbill** [comment]
Remote subtitles would be dealing with downloading subtitles from providers. Not downloading existing local subtitles. It would be best to open a [meta discussion](https://github.com/jellyfin/jellyfin-meta/discussions) for this.

**Comment by David-Moreira** [comment]
I'm sorry I do not understand your point. From the moment they are visible in the available subtitles screen I posted they are local and have been downloaded by jellyfin from a provider is that not the case? It's only a matter of downloading the local external. Subtitle.

**Comment by amdnsr** [comment]
@David-Moreira 
Thanks for bringing this up.

This is exactly the problem I too am facing, and is also in the feature requests here at https://features.jellyfin.org/posts/1316/download-locally-available-subtitle

This would be a great feature to have, especially when I can easily download the mp4 file, but downloading the subtitle files is currently not possible, and I have to search internet for the same subtitles that are already in my jellyfin server.

**Comment by David-Moreira** [comment]
Yea @amdnsr seems like it'd be an easy win to get this implemented, but I didn't get a satisfactory answer or even good feedback from jellyfin team... they must have their hands full... oh well... 
I was about to just fork it and implement it for myself, but I got pulled into other projects in the meantime.
I dislike having to use forks of the official projects, but I'll let you know If I happen to implement it.

----

> This would be a great feature to have, especially when I can easily download the mp4 file, but downloading the subtitle files is currently not possible, and I have to search internet for the same subtitles that are already in my jellyfin server.

The files should also be in the same folder as your media, if you have easy access to the server you can also just pull from there instead of searching the internet.

**Comment by amdnsr** [comment]
Yes, when I fiddled around with the network tab in chrome inspector, I found that it makes a call like the following when I pick a subtitle.
`http://localhost:8096/Videos/video-id-with-dashes/videoidwithoutdashes/Subtitles/5/0/Stream.js?api_key=<api_key>
`
will get you the json of the subtitle number 5.
It will not give you the name of the subtitle though, but still it is better than nothing, and suffices for most purposes.
